can/util.py

```diff
@@ -226,6 +226,24 @@ def _create_bus_config(config: Dict[str, Any]) -> typechecking.BusConfig:
         if not 0 < port < 65535:
             raise ValueError("Port config must be inside 0-65535 range!")
 
+    try:
```

Comment:
That's a really good point. The `can.Bus` docstring says
```
Instantiates a CAN Bus of the given interface, falls back to reading a configuration file from default locations.
```

But if the user tries to instantiate a Bus with just `can.Bus(interface="vector", channel=0, bitrate=500_000)`, the bitrate argument might be ignored if there is a complete set of `BitTimingFd` parameters in the configuration. So "falls back", as it says in the docs, is not really true anymore.

However, the same is true, if for example there is a `fd=True` entry in the configuration.

I'm not sure how to handle this. Would it be sufficient to update the `can.Bus` docs and add a `load_config=True` parameter? Then the user has the option to ignore config data completely.

Reply:
I opened #1474 for this.

Reply:
Do you want to keep the parameter values intended for BitTiming separate from the ones passed directly to the constructor of the interface? If so, another option would be to add a prefix (such as `bt_`) to the parameters that will make up the arguments to `BitTiming` or `BitTimingFd` in `Bus.__new__`. Then you could put them into the config file without interfering with the regular arguments.

Reply:
Do you have any scenario in mind, where a prefix would help?

Reply:
My original concern was towards interfaces that require a full set of CAN-FD parameters in the constructor. For the PCAN interface, you specify `f_clock,  nom_brp, data_brp, nom_tseg1, data_tseg1, ...`. In this special case, there is no complete overlap in parameter names between the regular constructor arguments and the `BitTimingFd` arguments. But if that was the case, you would have implicit behavior that is unexpected to the user: the `Bus.__new__` code detects that the parameters match the requirements of `BitTimingFd` and constructs the instance which then takes precedence.  I believe that there is no interface for which such a full overlap between `BitTiming[Fd]` parameters and regular constructor parameters exists, so I'm being overly cautious about this. Adding a prefix to the parameter names would solve this ambiguity, but it's not strictly necessary since it doesn't come into play at the moment.

Reply:
Let's assume there is a complete overlap and a timing object is constructed and takes precedence. The timing object and the paramters should be equivalent in that case, so it would make no difference, right?

Reply:
That is correct. Let's leave it like it is.